### DISCUSSION TITLE
请问大佬们哪里有较新的案例demo?

### DISCUSSION THREAD

**Discussion by ochive** [sofastack/sofa-ark#904] [Q&A] [answered]
[官方的文档](https://www.sofastack.tech/projects/sofa-boot/sofa-ark-spring-boot-demo/)照了撸了一遍没有成功, 想找demo学习下.
但文档里提到的demo似乎已经不维护了, 新的地址跳过去也是404.
https://github.com/sofastack-guides/sofa-ark-spring-guides

请大佬们给个资源🎉🎉🎉

**Comment by lvjing2** [comment, marked as answer]
https://github.com/koupleless/koupleless

koupleless.io

**Comment by lvjing2** [comment]
原文档里的链接也更新了

**Comment by ochive** [comment]
谢谢你，抱歉到现在才回复。
